src/ibex_bluesky_core/callbacks/__init__.py

```diff
@@ -32,6 +32,7 @@
 
 
 __all__ = [
+    "ChainedLiveFit",
```

Comment:
API thought: this is quite specific to polarisation measurements which are split by wavelength band. It might belong in a subpackage rather than the top level `callbacks` module.